temporalio/lib/temporalio/priority.rb

```diff
@@ -77,7 +77,7 @@ def self.default
     # @param priority_key [Integer, nil] The priority key
     # @param fairness_key [String, nil] The fairness key
     # @param fairness_weight [Float, nil] The fairness weight
-    def initialize(priority_key:, fairness_key: nil, fairness_weight: nil)
+    def initialize(priority_key: nil, fairness_key: nil, fairness_weight: nil)
```

Comment:
Python doesn't default to `None` at https://github.com/temporalio/sdk-python/blob/3c6fae6606428bb895c4a0df0c0a9c20c6e4bf82/temporalio/common.py#L985

Reply:
I'll fix it as well